test/e2e/service_test.go

```diff
@@ -106,38 +106,50 @@ func (data *TestData) createClusterIPServiceAndBackendPods(t *testing.T, name st
 	return svc, cleanup
 }
 
-// TestNodePortWindows tests NodePort Service on Windows Node. It is a temporary test to replace upstream Kubernetes one:
-// https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/test/e2e/windows/service.go#L42
-// Issue: https://github.com/antrea-io/antrea/issues/2289
-func TestNodePortWindows(t *testing.T) {
-	skipIfNoWindowsNodes(t)
-
+// TestNodePort tests traffic from Pods to NodePort Service.
+func TestNodePort(t *testing.T) {
 	data, err := setupTest(t)
 	if err != nil {
 		t.Fatalf("Error when setting up test: %v", err)
 	}
 	defer teardownTest(t, data)
 
+	nodeIP := clusterInfo.nodes[0].ip()
+	if net.ParseIP(nodeIP).To4() == nil {
+		nodeIP = fmt.Sprintf("[%s]", nodeIP)
+	}
+	t.Run("Windows NodePort Service", func(t *testing.T) {
+		// Windows NodePort Service: It is a temporary test to replace upstream Kubernetes one:
+		// https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/test/e2e/windows/service.go#L42
+		// Issue: https://github.com/antrea-io/antrea/issues/2289
```

Comment:
I thought this was fixed in the latest version of OVS Windows (which we use for CI testing)?